src/itemset.js

```diff
@@ -1,3 +1,16 @@
+/**
```

Comment:
Yes, this is a pretty good summary of itemsets.

As for the array, I think it's because it was easiest to implement by iterating over the child mugs and serializing each of them; other parts of the code make sure there is only ever one. I don't think we have ever tested or formally supported a dynamic select question with multiple itemsets.